dom/browser-element/mochitest/browserElement_Viewmode.js

```diff
@@ -0,0 +1,71 @@
+/* Any copyright is dedicated to the public domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Test that the onmozbrowsermetachange event for viewmode works.
```

Comment:
technically, the event is called `mozbrowsermetachange`. `onmozbrowsermetachange` is the name of the event listener function.